core/src/ancestor_hashes_service.rs

```diff
@@ -1227,9 +1225,12 @@ mod test {
 
             // Set up blockstore for responses
             let blockstore = Arc::new(Blockstore::open(&ledger_path).unwrap());
-            // Create slots [slot, slot + MAX_ANCESTOR_RESPONSES) with 5 shreds apiece
-            let (shreds, _) =
-                make_many_slot_entries(slot_to_query, MAX_ANCESTOR_RESPONSES as u64, 5);
+            // Create slots [slot - MAX_ANCESTOR_RESPONSES, slot) with 5 shreds apiece
```

Comment:
This test setup actually wasn't working before, would only send a truncated ancestor response starting from `slot_to_query`